### PR TITLE
Hotfix/fix scale

### PR DESCRIPTION
**Pull request by pNagi**

### CODE REVIEW

File: src/shared/nt-results/Link.react.js [hunk 1]
```diff
@@ -40,56 +34,64 @@ export const GET_DATA = {
...
 }
 
+// <text
+//   dx={source.x + (target.x - source.x) / 2}
+//   dy={source.y + (target.y - source.y) / 2}
+//   textAnchor='middle'
+// >
+//   {count}
+// </text>
+
```
Comment (0xchimz) [inline src/shared/nt-results/Link.react.js]: remove comment

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 2]
```diff
@@ -55,16 +57,45 @@ class ResultsByLoadContainer extends React.Component {
...
+
+  withInputOutputCount = nodes => nodes.map(node => {
+    const inputCount = node.type === 'load' ? node.count
+      : _.reduce(this.props.data.graph.links, (prev, link) => {
+        if (link.target === node._id && link.source !== node._id) {
+          return prev + link.count
+        }
+        return prev
+      }, 0)
+    const outputCount = _.reduce(this.props.data.graph.links, (prev, link) => {
```
Comment (0xchimz) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]: I think this line is too long. 
```javascript
withInputOutputCount = nodes => nodes.map(node => {
  const { graph } = this.props.data
  const inputCount = node.type === 'load' ? node.count
    : _.reduce(graph.links, (prev, link) => {
    ...
  const outputCount = _.reduce(graph.link, (prev, link) => {
```

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 3]
```diff
@@ -55,16 +57,45 @@ class ResultsByLoadContainer extends React.Component {
...
+      <div className={styles.nt}>
         {this.props.data.loading ? (
           <Loading key='loading' />
         ) : (
           <ResultsByLoad
-            nodes={_.cloneDeep(this.props.data.graph.nodes)}
+            nodes={this.withInputOutputCount(this.props.data.graph.nodes)}
             links={_.cloneDeep(this.props.data.graph.links)}
             paths={_.cloneDeep(this.props.data.graph.paths)}
+            totalInputCount={this.calculateTotalInputCount(this.props.data.graph.nodes)}
```
Comment (0xchimz) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]:
```javascript
const { loading, graph } = this.props.data
...
nodes={this.withInputOutputCount(graph.nodes)}
...
totalInputCount={this.calculateTotalInputCount(graph.nodes)}
```

File: src/shared/nt-results/ResultsByLoadContainer.react.js (hunk 2 above)
Comment (pNagi) [reply src/shared/nt-results/ResultsByLoadContainer.react.js]: the line is not too long. I think you mean the `this.props.data.graph.links`.

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 4]
```diff
@@ -86,16 +87,17 @@ class ResultsByLoadContainer extends React.Component {
   })
 
   render () {
+    const data = this.props.data
     return (
       <div className={styles.nt}>
         {this.props.data.loading ? (
```
Comment (0xchimz) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]: `data.loading`?

**Review by 0xchimz — CHANGES_REQUESTED** [review]

**Review by 0xchimz — APPROVED** [review]

**Pull request merged**